compare-sharding.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/bash
```

Comment:
I would prefer this script not to be on the root of the repo. How about the `tests` directory instead?